pkg/controller/migcluster/validation.go

```diff
@@ -267,13 +256,14 @@ func (r *ReconcileMigCluster) validateSaTokenPrivileges(cluster *migapi.MigClust
 	if err != nil {
 		return liberr.Wrap(err)
 	}
+
 	if !migrationSar.Status.Allowed || !veleroSar.Status.Allowed {
 		cluster.Status.SetCondition(migapi.Condition{
 			Type:     SaTokenNotPrivileged,
 			Status:   True,
 			Reason:   Unauthorized,
 			Category: Critical,
-			Message:  SaTokenNotPrivilegedMessage,
+			Message:  fmt.Sprintf("The `saToken` has insufficient privileges in either `migration.openshift.io` or `velero.io` group."),
```

Comment:
I'm concerned that by listing the API groups is opening a can of worms since it adds a little more information but is not a complete description of the auth needed.  Also, if the group name change, this would be easily missed.  I think it's better to leave the message as it was.

Reply:
Updated the `Message`.